db/migrate/20210510130434_create_participant_events.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class CreateParticipantEvents < ActiveRecord::Migration[6.1]
+  def change
+    create_table :participant_events, id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+      t.string :item_type, null: false
+      t.string :event, null: false
+      t.string :whodunnit
+      t.json :object
+      t.json :object_changes
+      t.datetime :created_at
+      t.uuid :item_id, default: -> { "gen_random_uuid()" }, null: false
```

Comment:
what is `item`? And do we really want a random uid as a default?

Reply:
Well, I imagine the `uid` in there comes from 
```
class ChangeItemIdTypeToUuid < ActiveRecord::Migration[6.1]
  def change
    remove_column :versions, :item_id, :bigint
    remove_index :versions, column: %i[item_type item_id], if_exists: true
    add_column :versions, :item_id, :uuid, default: "gen_random_uuid()", null: false
    add_index :versions, %i[item_type item_id]
  end
end
```
In that migration we were changing a field on an existing table, so not having a default was hard. In your case you are adding a new table, so not having a default should be easy, or so I hope.

Reply:
```
add_column :versions, :item_id, :uuid, default: "gen_random_uuid()", null: false
```
This doesn't seem right to me; the `item_id` for paper_trail is as I understand it a reference to the primary key field of whichever entity is being tracked. As such having a randomly generated guid would mean the data would be corrupt because it wouldn't point to any valid primary key. I'm inclined to ignore the migration you have quoted and just make our own one correct.

Reply:
I would also be inclined to make your migration correct. 

I mentioned that migration because that's how I think the random uid got into schema, which you copied to get your migration in the first place. Which I really think you shouldn't need or do.

Reply:
Sure. Dilemma that I had was that it was originally a bigint and we seem to have moved away from those. It won't ever take the default, since the value will be filled in on creation in any case. 

Something that I think is way more important is that it does seem to suggest that the production papertrail may have been broken by that migration which I think is a much bigger issue that how we define a default (that is never actually going to be used). Unless you're not actually using papertrail for anything, or it wasn't used for anything prior to the migration. I'd be very concerned about the data integrity if it's being relied on for anything.

I'll remove the default here, but keep the uid column type.